docs/aks-installation.md

```diff
@@ -20,18 +20,18 @@
 
 ## Prerequisites
 
-1. Install and configure [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
+1. Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) v1.24+.
 2. Install [Terraform](https://learn.hashicorp.com/terraform/getting-started/install.html). Recommend v1.2.2.
 3. Install `jq`, `pv`, and `bzip2`.
```

Comment:
Please validate if all this is still require

Reply:
For aws its all required, for azure seems bzip2 is no longer required